types/async-redis/async-redis-tests.ts

```diff
@@ -0,0 +1,56 @@
+import asyncRedis from "async-redis";
```

Comment:
should be probably:
```suggestion
const asyncRedis = require("async-redis");
```
as per CommonJS practise (valid until 3.9)
https://devblogs.microsoft.com/typescript/announcing-typescript-3-9-rc/#cjs-auto-imports

Reply:
Is it really necessary? I have undone this commit because of tslint error "no-var-required"

I would like to avoid of editing config files

Reply:
my fualt, sorry, it should read:
```suggestion
import asyncRedis =  require("async-redis");
```